backend/app/core/dependencies.py

```
@@ -0,0 +1,25 @@
+"""
+依存性注入の設定
+"""
+from typing import AsyncGenerator
```

Ruff (0.11.9)

`typing.AsyncGenerator` imported but unused

Remove unused import: `typing.AsyncGenerator`

(F401)